cmd/terramate/e2etests/run_test.go

```diff
@@ -1836,8 +1835,8 @@ func TestRunContinueOnError(t *testing.T) {
 
 	cli := newCLI(t, s.RootDir())
 	assertRunResult(t, cli.run("run", "cat", "main.tf"), runExpected{
-		IgnoreStderr: true,
-		Status:       1,
+		StderrRegex: "one or more commands failed",
```

Comment:
this test was not behaving correctly. While testing this PR, I forced a panic() in the core Terramate and this test passed with the panic.